core/src/luigi-config/navigation/navigation-data-init.js

```diff
@@ -221,6 +223,35 @@ async function fetchNavigationData(authData, permissionSet) {
   }
 }
 
+async function getObservabilityNodes(authData, enabledFeatures) {
```

Comment:
can we move the logic to the `features.js` file?

Reply:
But it's mostly about adding nodes